tests/qubit_codes/test_surface_code.py

```diff
@@ -23,6 +23,7 @@
 import numpy as np
 from numpy.random import default_rng as rng
 import pytest
+from sympy import intersection
```

Comment:
What is the purpose in using sympy's intersection?

Reply:
None — I blame the auto-import feature of my editor. Good catch! 👁️